<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CAL3180" ignored="0" name="Default test" tests="21" failures="3" timestamp="2022-05-30T10:48:59 IST" time="83.513" errors="0">
  <testcase name="devLoginInitiate" time="67.198" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <failure type="org.openqa.selenium.NoSuchWindowException" message="no such window: window was already closed
  (Session info: chrome=101.0.4951.67)
Build info: version: &amp;apos;3.4.0&amp;apos;, revision: &amp;apos;unknown&amp;apos;, time: &amp;apos;unknown&amp;apos;
System info: host: &amp;apos;CAL3180&amp;apos;, ip: &amp;apos;10.10.3.180&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_161&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{networkConnectionEnabled=false, chrome={chromedriverVersion=101.0.4951.41 (93c720db8323b3ec10d056025ab95c23a31997c9-refs/branch-heads/4951@{#904}), userDataDir=C:\Users\SANDEE~1\AppData\Local\Temp\scoped_dir11780_931222040}, timeouts={implicit=0.0, pageLoad=300000.0, script=30000.0}, pageLoadStrategy=normal, unhandledPromptBehavior=dismiss and notify, strictFileInteractability=false, platform=ANY, proxy=Proxy(), goog:chromeOptions={debuggerAddress=localhost:61844}, webauthn:extension:credBlob=true, acceptInsecureCerts=false, browserVersion=101.0.4951.67, browserName=chrome, javascriptEnabled=true, platformName=windows, setWindowRect=true, webauthn:extension:largeBlob=true, webauthn:virtualAuthenticators=true}]
Session ID: 17cf10c3c6bc8851907b363b86398081">
      <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: window was already closed
  (Session info: chrome=101.0.4951.67)
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'CAL3180', ip: '10.10.3.180', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{networkConnectionEnabled=false, chrome={chromedriverVersion=101.0.4951.41 (93c720db8323b3ec10d056025ab95c23a31997c9-refs/branch-heads/4951@{#904}), userDataDir=C:\Users\SANDEE~1\AppData\Local\Temp\scoped_dir11780_931222040}, timeouts={implicit=0.0, pageLoad=300000.0, script=30000.0}, pageLoadStrategy=normal, unhandledPromptBehavior=dismiss and notify, strictFileInteractability=false, platform=ANY, proxy=Proxy(), goog:chromeOptions={debuggerAddress=localhost:61844}, webauthn:extension:credBlob=true, acceptInsecureCerts=false, browserVersion=101.0.4951.67, browserName=chrome, javascriptEnabled=true, platformName=windows, setWindowRect=true, webauthn:extension:largeBlob=true, webauthn:virtualAuthenticators=true}]
Session ID: 17cf10c3c6bc8851907b363b86398081
at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:150)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:115)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:45)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:82)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:637)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:694)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.parentFrame(RemoteWebDriver.java:985)
at com.Automation.Base.ActionEngine.switchToParentFrame(ActionEngine.java:523)
at com.Automation.Pages.DEV_LoginInitiation.devLoginInitiation(DEV_LoginInitiation.java:222)
at com.Automation.Testcases.Dev_PositiveFlow.devLoginInitiate(Dev_PositiveFlow.java:152)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- devLoginInitiate -->
  <testcase name="devLgnRevReturn" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- devLgnRevReturn -->
  <testcase name="devLgnReinitiate" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- devLgnReinitiate -->
  <testcase name="devLgnReview" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- devLgnReview -->
  <testcase name="devLgnApprReturn" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- devLgnApprReturn -->
  <testcase name="devLgnAgnReinitiate" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- devLgnAgnReinitiate -->
  <testcase name="devLgnAgnReview" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- devLgnAgnReview -->
  <testcase name="devLgnAprArrove" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- devLgnAprArrove -->
  <testcase name="dev_Otd" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- dev_Otd -->
  <testcase name="dev_Qad" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- dev_Qad -->
  <testcase name="doc_RevReturn" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- doc_RevReturn -->
  <testcase name="doc_ReInitiate" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- doc_ReInitiate -->
  <testcase name="doc_LgnReview" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- doc_LgnReview -->
  <testcase name="doc_LgnApprove" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- doc_LgnApprove -->
  <testcase name="doc_TaskAllocation" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- doc_TaskAllocation -->
  <testcase name="doc_TaskExecution" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- doc_TaskExecution -->
  <testcase name="doc_Closure" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- doc_Closure -->
  <testcase name="dev_Closure" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- dev_Closure -->
  <testcase name="dev_ClosureReturn" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <skipped/>
  </testcase> <!-- dev_ClosureReturn -->
  <testcase name="dev_ClosureReInitiate" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <failure type="org.testng.TestNGException" message="
Method public void com.Automation.Testcases.Dev_PositiveFlow.dev_ClosureReInitiate(java.lang.String,java.lang.String) requires a @DataProvider named : DevClsRei">
      <![CDATA[org.testng.TestNGException: 
Method public void com.Automation.Testcases.Dev_PositiveFlow.dev_ClosureReInitiate(java.lang.String,java.lang.String) requires a @DataProvider named : DevClsRei
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- dev_ClosureReInitiate -->
  <testcase name="dev_ClosureApprove" time="0.0" classname="com.Automation.Testcases.Dev_PositiveFlow">
    <failure type="org.testng.TestNGException" message="
Method public void com.Automation.Testcases.Dev_PositiveFlow.dev_ClosureApprove(java.lang.String) requires a @DataProvider named : DevClsAppr">
      <![CDATA[org.testng.TestNGException: 
Method public void com.Automation.Testcases.Dev_PositiveFlow.dev_ClosureApprove(java.lang.String) requires a @DataProvider named : DevClsAppr
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- dev_ClosureApprove -->
</testsuite> <!-- Default test -->
